tests/core/lowering/test_module_level_fallback.cpp

```diff
@@ -0,0 +1,142 @@
+#include <string>
+#include <unordered_set>
+#include "core/compiler.h"
+#include "core/lowering/lowering.h"
+#include "gtest/gtest.h"
+#include "tests/util/util.h"
+#include "torch/script.h"
+
+TEST(Lowering, LowerResNet18ModuleFallbackCorrectly) {
+  torch::jit::script::Module mod;
+  try {
+    mod = torch::jit::load("tests/modules/resnet18_traced.jit.pt");

```

Comment:
for this sort of test we need a way similar model, we should basically have 1-2 ops in the !to_compile set